config/jobs/kubernetes/sig-node/containerd.yaml

```diff
@@ -810,7 +810,7 @@ periodics:
       - --gcp-nodes=4
       - --gcp-zone=us-west1-b
       - --ginkgo-parallel=30
-      - --image-family=cos-89-lts
+      - --image-family=cos-101-lts
```

Comment:
(for future), does it make sense to just use `cos-stable` here so we don't need to update this everytime?

Reply:
Makes sense, but I'd be worried that tests break if something changes between COS versions (such as the cgroups v2 directive). Still, it's probably worth the trade.